.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         java: [ 11, 17, 21 ]
         os: [ubuntu-latest,windows-latest]
-        bwc_version : ["2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0"]
+        bwc_version : ["2.9.0","2.10.0","2.11.0","2.12.0","2.13.0-SNAPSHOT","2.14.0"]
```

Comment:
snapshots are prone to change and when it changes, it might impact the test results, do we really need this?

Reply:
As the BWC test issue is going to be resolved, there is no need to merge this PR.

Reply:
This PR is experimenting whether the BWC tests can get passed with 2.13.0-SNAPSHOT